addons/binding/org.openhab.binding.tellstick/README.md

```diff
@@ -0,0 +1,95 @@
+
+## Discovery
+
+Both Telldus Core and Live devices will be discovered once the bridges are configured.
+
+## Thing Configuration
+```
+Bridge tellstick:telldus-core:1 "Tellstick Duo" []
+Bridge tellstick:telldus-live:2 "Tellstick ZWave" [publicKey="XXX", privateKey="YYYY", token= "ZZZZ", tokenSecret="UUUU"]
+```
+## Channels
+=======
+There is two levels of discovery. When you add this binding it will try to discover the Telldus Core Bridge (this only works if you have Telldus installed and right 32/64 bit version of java).
+If you want to use the Telldus Live bridge, then you manually need to add this thing.
+When you add either of those all devices and sensors will be discovered and reported in the inbox.
+
+## Thing Configuration
+
+#####Telldus Core Bridge#####
+- **libraryPath:** The path to tellduscore.dll/so, 
+- **resendInterval:** The interval between each transmission of command, default 100ms.
+
+#####Telldus Live Bridge#####
+To configure Telldus Live you have request OAuth tokens from Telldus. Goto this page
+http://api.telldus.com/keys/index and request your keys and update the config.
+- **privateKey:** Private key
+- **publicKey:** Public key
+- **token:** Token
+- **tokenSecret:** Token secret
+- **refreshInterval:** How often we should contact Telldus.Live to check for updates
+
+The devices and sensors should not be configured by hand, let the discovery/inbox configure these.
+
+## Channels
+
+Actuators (dimmer/switch) support the following channels:
+| Channel Type ID | Item Type    | Description  |
+|-----------------|--------------|----------------------------------------------- |
+| dimmer          | Number       | This channel indicates the current dim level |
+| state           | Switch       | This channel indicates whether a device is turned on or off. |
+| timestamp       | DateTime     | This channel reports the last time this device state changed. |
+
+Sensors support the following channels:
+| Channel Type ID | Item Type    | Description  |
+|-----------------|--------------|----------------------------------------------- |
+| humidity        | Number       | This channel reports the current humidity in percentage |
+| temperature      | Number       | This channel reports the current temperature in celsius |
+| timestamp       | DateTime     | This channel reports the last time this sensor was updates. |
+## Full Example
+```xtend
+Bridge tellstick:telldus-core:1 "Tellstick Duo" []
+Bridge tellstick:telldus-live:2 "Tellstick ZWave" [refresh="10000", publicKey="XXXXXXXX", privateKey="YYYYYY", token= "ZZZZZZZZ", tokenSecret="UUUUUUUUUU"]
+```
```

Comment:
I don't recommend doing that manually as there is no way for the user to find the correct ID's. Do you still think I should add examples ?